spec/helpers/application_helper_spec.rb

```diff
@@ -92,4 +92,40 @@
       end
     end
   end
+
+  describe '#concurrent_events' do
+    before :each do
+      @other_event = create(:event, program: conference.program, state: 'confirmed')
+      @event_schedule = create(:event_schedule, event: event, start_time: conference.start_date, room: create(:room))
+      @other_event_schedule = create(:event_schedule, event: @other_event, start_time: conference.start_date, room: create(:room))
+    end
+
+    describe 'does return correct concurrent events' do
+      it 'when events starts at the same time' do
+        expect(concurrent_events(event).include?(@other_event)).to eq true
+      end
+
+      it 'when event is in between the other event' do
+        @event_schedule.update_attributes(start_time: rand(@other_event_schedule.start_time...@other_event_schedule.end_time))
+        expect(concurrent_events(event).include?(@other_event)).to eq true
+      end
+    end
+
+    describe 'does not return as concurrent event ' do
+      it 'when event is not scheduled' do
+        @event_schedule.destroy
```

Comment:
Sorry, doesn't https://github.com/openSUSE/osem/blob/master/spec/factories/event_schedule.rb#L17 create a selected schedule for the conference while creating `@event_schedule` in L99?